src/main/java/com/elixrlabs/doctorpatientmanagementsystem/constants/ApiConstants.java

```diff
@@ -6,6 +6,8 @@
 public class ApiConstants {
     public static final String DOCTORS_END_POINT = "/doctors";
     public static final String GET_DOCTOR_BY_ID = "/doctors/{id}";
+    public static final String PATIENT_API="/patient";
```

Comment:
Please format this line
```suggestion
    public static final String PATIENT_API = "/patient";
```

Reply:
done